roofit/roostats/inc/RooStats/RooStatsUtils.h

```diff
@@ -58,9 +58,9 @@ namespace RooStats {
    }
 
    /// Compute the Asimov Median significance for a Poisson process
-   /// with s = expected number of signal events, b = expected numner of background events
-   /// and optionally sigma_b = expected uncertainty of backgorund events   
-   Double_t AsimovSignificance(Double_t s, Double_t b, Double_t sigma_b = 0.0 ); 
+   /// with s = expected number of signal events, b = expected number of background events
```

Comment:
/// param[in] s = expected number of signal events
/// param[in] b = expected number of background events
/// param[in] sigma_b : expected uncertainty of background events. A default value of 0 is used if it is not given.